devicetypes/smartthings/zwave-door-window-sensor.src/zwave-door-window-sensor.groovy

```diff
@@ -192,9 +193,11 @@ def zwaveEvent(physicalgraph.zwave.commands.wakeupv1.WakeUpNotification cmd)
 	}
 	if (!state.lastbat || now() - state.lastbat > 53*60*60*1000) {
 		cmds << command(zwave.batteryV1.batteryGet())
-	} else {
-		cmds << zwave.wakeUpV1.wakeUpNoMoreInformation().format()
 	}
+	if (device.currentValue("contact") == null) { // Incase our initial request didn't make it
+		cmds << command(zwave.sensorBinaryV2.sensorBinaryGet(sensorType: zwave.sensorBinaryV2.SENSOR_TYPE_DOOR_WINDOW))
+	}
+	cmds << zwave.wakeUpV1.wakeUpNoMoreInformation().format()
```

Comment:
I'll go ahead and re-arrange the code a little so that the contact check happens first